bin.js

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env node
+const path = require('path');
 const main = require('./app.js');
-main();
+main({
+  pluginDirectories: [path.join(process.cwd(), 'tasks')]
```

Comment:
I don't understand what the need for this is. In `clientkit`, wouldn't it just be 

```
tasks:
  config: '{{CKDIR}}/tasks/config'
```